Application/EdFi.Ods.Admin/Security/IdentityProvider.cs

```diff
@@ -157,12 +160,12 @@ public bool ConfirmEmailWithToken(string email, string token)
         {
             using (var identityUserManager = GetIdentityUserManager())
             {
-                IdentityResult result = null;
+               
                 var identityUser = identityUserManager.FindByEmail(email);
 
                 if (identityUser != null)
-                {
-                    result = identityUserManager.ConfirmEmail(identityUser.Id, token);
+                {               
```

Comment:
remove extra whitespace

Reply:
Did not find any extra whitespace.

Reply:
the whitespace is still showing up, if you refomat the document it should clear it